data/functions.sql

```diff
@@ -189,12 +189,12 @@ BEGIN
             OR waterway_val IN ('lock')
             OR aerialway_val IN ('station')
             OR railway_val IN ('halt', 'tram_stop')
-            OR tourism_val IN ('alpine_hut'))
+            OR tourism_val IN ('alpine_hut')
+            OR amenity_val='prison')
```

Comment:
Most the others follow IN () pattern, should we do that here?

`OR amenity_val IN ('prison'))`